SweetUIKit.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   s.name             = "SweetUIKit"
   s.summary          = "Helpers and sugar for the UIKit framework."
-  s.version          = "1.13.1"
+  s.version          = "1.15.0"
```

Comment:
Why 0.15 instead of 0.14?

Reply:
Because the [most recent release](https://github.com/UseSweet/SweetUIKit/tree/1.14.0) was 1.14.0 - it just was never updated in the podspec.

Reply:
Ahhhh